doc/man3/SSL_get_handshake_rtt.pod

```diff
@@ -0,0 +1,60 @@
+=pod
+
+=head1 NAME
+
+SSL_get_handshake_rtt
+- get round trip time for SSL Handshake
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_get_handshake_rtt(const SSL *s, uint64_t *rtt);
+
+=head1 DESCRIPTION
+
+SSL_get_handshake_rtt() retrieves the round-trip time (RTT) for I<ssl>.
+
+This metric is represented in microseconds (us) as a uint64_t data type.
+
+=head1 NOTES
+
+This metric is created by taking two timestamps during the handshake and providing the
+difference between these two times.
+
+When acting as the server, one timestamp is taken when the server is finished
+writing to the client. This is during the ServerFinished in TLS 1.3 and 
+ServerHelloDone in TLS 1.2. The other timestamp is taken when the server is done reading 
+the client's response. This is after the client has responded with ClientFinished.
+
+When acting as the client, one timestamp is taken when the client is finished writing 
+the ClientHello and early data (if any). The other is taken when client is done reading the server's response.
+This is after ServerFinished in TLS 1.3 and after ServerHelloDone in TLS 1.2.
+
+In addition to network propagation delay and network stack overhead, this metric 
+includes processing time on both endpoints, as this is based on TLS protocol-level 
+messages and the TLS protocol is not designed to measure network timings. In some cases the 
+processing time can be significant, especially when the processing includes asymmetric cryptographic 
+operations.
+
+=head1 RETURN VALUES
+
+Returns 1 if the TLS handshake round-trip time is successfully retrieved.
+Returns 0 if the TLS handshake round-trip time is not yet determined or if an error occurred in 
```

Comment:
Please avoid lines with more than 80 chars and whitespace at line ends.
Ideally, place newlines after end `of` each sentence (full stop).

Reply:
Is this pointing out lines in the entire file? I wasn't sure when to split lines and when to not in the NOTES paragraphs.

I can chop this one line down to `Returns 0 if the TLS handshake round-trip time is not yet determined or in case of error.` which is 89 chars.

Reply:
Sorry that my "please avoid lines with more than 80 chars" was expressed misleadingly.
What I actually meant was: please split lines longer than 80 chars.